doc/source/whatsnew/v0.18.1.rst

```diff
@@ -471,30 +471,35 @@ New Behavior:
 Changes in ``read_csv`` exceptions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. ipython:: python
+   :suppress:
+
+   import io
```

Comment:
can you move the import to the beginning of the next code block, and remove this block.